### PR TITLE
Validate basic fixes

### PR DESCRIPTION
**Pull request by skosito**

### CODE REVIEW

File: baseledger/x/bridge/keeper/attestation_handler.go [hunk 1]
```diff
@@ -146,11 +146,12 @@ func (a AttestationHandler) Handle(ctx sdk.Context, att types.Attestation, claim
 		} else {
 			hash, _ := claim.ClaimHash()
 			a.keeper.Logger(ctx).Error("Invalid receiver address",
-				"cause", err.Error(),
```
Comment (skosito) [inline baseledger/x/bridge/keeper/attestation_handler.go:149]: err is nil here

File: baseledger/x/bridge/types/message_ubt_deposited_claim.go [hunk 2]
```diff
@@ -67,9 +67,6 @@ func (msg *MsgUbtDepositedClaim) ValidateBasic() error {
 	if _, err := sdk.AccAddressFromBech32(msg.Creator); err != nil {
 		return sdkerrors.Wrap(sdkerrors.ErrInvalidAddress, "orchestrator")
 	}
-	if _, err := sdk.AccAddressFromBech32(msg.BaseledgerReceiverAccountAddress); err != nil {
```
Comment (skosito) [inline baseledger/x/bridge/types/message_ubt_deposited_claim.go:70]: i dont think the rest of these should be removed from here, same for other claim, will follow up offline with this

File: tests/test/baseledger.test.js [hunk 3]
```diff
@@ -460,11 +460,11 @@ describe('add new node', async function() {
 
 describe('baseledger transaction', async function() {
   this.timeout(50000);
-  before(() => {
+  beforeEach(() => {
```
Comment (skosito) [inline tests/test/baseledger.test.js:463]: deposit from first test messes up with others, so to isolate them better

File: tests/test/baseledger.test.js [hunk 4]
```diff
@@ -613,6 +613,150 @@ describe('baseledger transaction', async function() {
...
+  this.timeout(70000);
+  before(() => {
+    startTestNet();
+  });
+
+  after(() => {
+    cleanTestNet();
+  });
+
+  it('should pass bad deposit address and not increase balance', async function() {
```
Comment (skosito) [inline tests/test/baseledger.test.js:626]: both tests start with bad input, and the with good, checking if event nonces recovered properly

**Review by ognjenkurtic — APPROVED** [review]

**Pull request merged**